sopel/cli/utils.py

```diff
@@ -230,13 +230,10 @@ def find_config(config_dir, name, extension='.cfg'):
         '/home/username/.sopel/extra.ini'
 
     """
+    if not name.endswith(extension):
```

Comment:
You changed the behavior, but you didn't upgrade the docstring, which says:

```
>>> utils.find_config(config.DEFAULT_HOMEDIR, 'local')
'/home/username/.sopel/local'
```

This must be modified by this:

```
>>> utils.find_config(config.DEFAULT_HOMEDIR, 'local')
'/home/username/.sopel/local.cfg'
```

The test you modified is a proof of behavior change.

Reply:
The old docstring was also incorrect :upside_down_face: The code would find ./extra.ini before $conf/extra.ini, and the local one would be abspath.

The whole searching pwd thing is kinda weird too... I can guess why we do it, but it feels fucky for anything but sopel .cfgs - should we make it only search pwd when `search_pwd=True` or something, so plugins will only get stuff within the homedir the main sopel config sets?

Reply:
> but it feels fucky for anything but sopel .cfgs

Fortunately that's all this function is supposed to be used for. This is `sopel.cli.utils`, not a module we expect plugin authors ever to touch.